index.html

```diff
@@ -1,20 +1,45 @@
-Static Site
+<!DOCTYPE html>
```

Comment:
every HTML file should begin with a DOCTYPE These are special tags that don't require a closing tag or a `/`. 

@see http://www.w3schools.com/tags/tag_doctype.asp